### DISCUSSION TITLE
Max bet size incorrect value

### DISCUSSION THREAD

**Discussion by ghost** [cloudbet/docs#56] [Q&A]
Hi,

I am having some issues with querying the `v2/odds/lines` endpoint on non-live games. I am using this endpoint to query the max bet size right before placing a bet, but sometimes (quite often actually) it returns an incorrect value. For example I get a 1000 value returned however the real max bet size is like 10. 
Same happens on the website where I can go onto that bet, it says max bet size is 1000. I try to place it and the value is then replaced with the actual max bet size. Is this a know issue? Anything I can do to avoid it?

Best regards

**Comment by kgravenreuth** [comment]
Hi @sonderangebot10 

Thanks for raising this issue. 

As mentioned in #55 , the limits are controlled by our risk management system and could be dynamically changed for your player. The limits could potentially be increased over time with more betting activity, but this is not a guarantee and depends on our risk management system's assessment.

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
